strawberryfields/ops.py

```diff
@@ -1652,10 +1647,10 @@ class GraphEmbed(Decomposition):
         A (array): an :math:`N\times N` complex or real symmetric matrix
         max_mean_photon (float): threshold value. It guarantees that the mode with
             the largest squeezing has ``max_mean_photon`` as the mean photon number
-            i.e., :math:`sinh(r_{max})^2 == max_mean_photon`
+            i.e., :math:`sinh(r_{max})^2 ==` max_mean_photon
```

Comment:
Likewise,
```suggestion
            i.e., :math:`sinh(r_{max})^2 = \text{max_mean_photon}`
```
It looks a bit odd having `max_mean_photon` outside the equation in normal font.